API's WITH DJango Views/companyapi/apiviews/views.py

```diff
@@ -0,0 +1,248 @@
+from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+from apiviews.models import *
+from apiviews.serializers import *
+from django.shortcuts import get_object_or_404
+from rest_framework.decorators import action
+
+@api_view(['GET','POST','PUT','PATCH','DELETE'])
+def company_api(request, pk=None):
```

Comment:
Please use viewsets here as instructed. This project was supposed to be completed using viewsets.

Reply:
This file will be reviewed further once updated.